lang/py/avro/test/test_schema.py

```diff
@@ -691,12 +890,28 @@ def test_fixed_decimal_invalid_max_precision(self):
     def test_parse_invalid_symbol(self):
         """Disabling enumschema symbol validation should allow invalid symbols to pass."""
         test_schema_string = json.dumps({"type": "enum", "name": "AVRO2174", "symbols": ["white space"]})
-        with self.assertRaises(avro.errors.InvalidName, msg="When enum symbol validation is enabled, an invalid symbol should raise InvalidName."):
-            avro.schema.parse(test_schema_string, validate_enum_symbols=True)
+
         try:
-            avro.schema.parse(test_schema_string, validate_enum_symbols=False)
-        except avro.errors.InvalidName:  # pragma: no coverage
-            self.fail("When enum symbol validation is disabled, an invalid symbol should not raise InvalidName.")
+            case = avro.schema.parse(test_schema_string, validate_enum_symbols=True)
+        except avro.errors.InvalidName:
+            pass
+        else:
+            self.fail("When enum symbol validation is enabled, " "an invalid symbol should raise InvalidName.")
+
+        try:
+            case = avro.schema.parse(test_schema_string, validate_enum_symbols=False)
```

Comment:
According to code scanning, `case` is never used.

Reply:
Removed 👍